ambry-api/src/main/java/com.github.ambry/messageformat/BlobProperties.java

```diff
@@ -23,17 +23,24 @@
  */
 public class BlobProperties {
 
-  protected long blobSize;
-  protected String serviceId;
-  protected String ownerId;
-  protected String contentType;
-  protected boolean isPrivate;
-  protected long timeToLiveInSeconds;
-  protected long creationTimeInMs;
+  public static final short ACCOUNTID_DEFAULT_VALUE = -1;
```

Comment:
can we change this to something other than "default" - this is more of invalid than default is my understanding.